site/content/en/docs/tutorials/ambassador_ingress_controller.md

```diff
@@ -23,7 +23,7 @@ configured via both, [Ingress](https://kubernetes.io/docs/concepts/services-netw
 ### Installing Ambassador
 
 Ambassador is available as a Minikube
-[addon]((https://github.com/kubernetes/minikube/tree/master/deploy/addons/ambassdor)). Install it by running -
+[addon](https://minikube.sigs.k8s.io/docs/commands/addons/). Install it by running -
```

Comment:
same as these, change them to "Ref" links

Reply:
Incorporated the comments